### PR TITLE
Add GitHub Action workflows

### PR DESCRIPTION
**Pull request by cwdavies**
This adds GitHub Action workflows for frontend/backend and to clean up Action storage artifacts.

We use Actions storage to store coverage files from the frontend/backend test jobs to be consumed by the coverage jobs. This storage is limited to 500MB. Once the coverage jobs run there's no need to keep the coverage files around. This workflow will run every hour and remove all artifacts from completed jobs.

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Review by willbarton — CHANGES_REQUESTED** [review]
I think we should consider the level of complexity we actually need here rather than just importing the cfgov-refresh setup wholesale.

We could probably combine the frontend and backend and coverage into a single job, or a single workflow with multiple jobs, I think.

I also think we should also figure out why Travis is failing before we try to replace it.

File: tox.ini [hunk 1]
```diff
@@ -27,21 +28,21 @@ deps=
...
-ignore = 
+ignore =
     # Allow assigning lambda expressions
     E731,
     # Allow line breaks after binary operators
     W503,
 
 exclude =
-    # Some of this are excluded for speed of directory traversal. Not all of 
+    # Some of this are excluded for speed of directory traversal. Not all of
```
Comment (higs4281) [inline tox.ini:45]:
```suggestion
    # Some of these are excluded for speed of directory traversal. Not all of
```

File: .github/workflows/artifact_cleanup.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Our unittest action uses GitHub artifacts to store coverage files from the 
+# frontend and backend jobs for consumption by the coverage jobs. This cleans 
+# up old artifact files that we don't know once the coverage jobs have run.
+# https://github.com/marketplace/actions/purge-artifacts 
+name: 'Artifact cleanup'
```
Comment (willbarton) [inline .github/workflows/artifact_cleanup.yml:5]: We don't need this job if we're not storing coverage files off between jobs right now.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+        env:
+          RUNNER_TEMP: /tmp
```
Comment (willbarton) [inline .github/workflows/ci.yml:18]: Do we need this in GitHub Actions or was it only for act?

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+          python-version: 3.6
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+
+      - name: Run back-end tests
+        run: |
+          rm -rf /github/workspace/.tox/
```
Comment (willbarton) [inline .github/workflows/ci.yml:83]: Is this necessary? It isn't necessary in any of the other repos we're using Actions to run tox.

File: .github/workflows/artifact_cleanup.yml (hunk 2 above)
Comment (Scotchester) [reply .github/workflows/artifact_cleanup.yml:5]: Yeah, never finished my commit to clean this up last week. Completing that now.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (Scotchester) [reply .github/workflows/ci.yml:18]: Can't remember for sure. Will try taking it out.

File: .github/workflows/ci.yml (hunk 4 above)
Comment (Scotchester) [reply .github/workflows/ci.yml:83]: Can't remember for sure if it was only for Act. Will try taking it out.

**Review by willbarton — APPROVED** [review]
Let's squash it and ship it!

**Review by higs4281 — APPROVED** [review]

**Pull request merged**